rts/Lua/LuaFonts.cpp

```diff
@@ -205,6 +207,49 @@ int LuaFonts::DeleteFont(lua_State* L)
 	return meta_gc(L);
 }
 
+/*** Adds a fallback font for the font rendering engine.
+ *
+ * Fonts added first will have higher priority.
+ * When a glyph isn't found when rendering a font, the fallback fonts
+ * will be searched first, otherwise os fonts will be used.
+ *
+ * The application should listen for the unsynced 'FontsChanged' callin so
+ * modules can clear any already reserved display lists or other relevant
+ * caches.
+ *
+ * Note the callin won't be executed at the time of calling this method,
+ * but later, on the Update cycle (before other Update and Draw callins).
+ *
+ * @function gl.AddFallbackFont
+ * @string filePath VFS path to the file, for example "fonts/myfont.ttf"
```

Comment:
Does this mean font files not in the VFS (i.e. loose files somewhere on user disk aka VFS.RAW) can't be used? If they can be used, how do I specify which one to use if two are under the same path (one raw and one in the vfs)?

Reply:
Well, tbh I don't know XD. It goes through the same path other fonts use to load as well, basically doing: `FileHandler f(fontPath)`.

Reply:
If we're not (yet) letting people specify their own VFS mode then add a @remark that it uses the RAW_FIRST vfs mode.

Reply:
done.

Reply:
well, didn't use a specific @ tag since I don't see any that can be used here, just added it to the comment for the @param.